CommonLibSF/.editorconfig

```diff
@@ -0,0 +1,11 @@
+[*]
+charset = utf-8
+insert_final_newline = true
+
+[*.{asm,c,cc,cpp,cxx,h,hpp,hxx,inc,inl,ixx}]
+indent_style = tab
```

Comment:
pls no tabs 😭

Reply:
okay
    changing this would result a mass reformatting from workflow, I'll wait until we have a final style guide for this then

Reply:
Sounds good, let's leave that for a post-merge maintenance commit